### PR TITLE
Create_Resoure_Detail_User

### PR DESCRIPTION
**Pull request by ghost**
## Description
Create Resoure Detail User.
## Migrations
 NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

- app/Http/Controllers/
- routes/web.php

### CODE REVIEW

**Comment by at-huydq** [comment]
LGTM.

File: routes/web.php [hunk 1]
```diff
@@ -14,3 +14,9 @@
 Route::get('/', function () {
     return view('welcome');
 });
+
+
+//Admin 
+Route::group(['prefix' => 'admin', 'namespace' => 'Admin'], function () {
+   Route::resource('user', 'UserController', ['except' => ['create', 'store']]);
```
Comment (at-xinhnguyen) [inline routes/web.php]: use `users` instead of `user`

**Comment by at-huydq** [comment]
LGTM.

**Comment by at-huydq** [comment]
lgtm.

**Pull request merged**